Apps/WebClient/src/ClientApp/src/services/restConfigService.ts

```diff
@@ -5,8 +5,10 @@ import { IConfigService, IHttpDelegate, ILogger } from "@/services/interfaces";
 import ErrorTranslator from "@/utility/errorTranslator";
 
 export class RestConfigService implements IConfigService {
-    private readonly CONFIG_BASE_URI: string = "/configuration";
     private logger;
+    private readonly CONFIG_BASE_URI: string = `${
+        import.meta.env.VITE_CONFIG_BASE_URI || ""
+    }/configuration`;
     private http;
```

Comment:
To mirror the other services, I'd suggest separating into 2 properties.

```
    private readonly CONFIG_BASE_URI = "configuration";
    private logger;
    private http;
    private baseUri = import.meta.env.VITE_CONFIG_BASE_URI || "/";
```

This would require a change in the getConfiguration() function:

```
            .getWithCors<ExternalConfiguration>(`${this.baseUri}${this.CONFIG_BASE_URI}/`)
```

And a change to the .env files to include a trailing slash.

Since this is essentially the endpoint for the WebClient server, maybe the environment variable should be named WEB_CLIENT_BASE_URI instead of CONFIG_BASE_URI?

Reply:
I've updated the variable name but it has to be prefixed with VITE_ to be exposed.